ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -408,6 +409,11 @@ instance (ByronGiven, Typeable cfg, ConfigContainsGenesis cfg)
 
   computeGenTxId = ByronTxId . Crypto.hash . CC.UTxO.taTx . unByronTx
 
+  computeGenTxSize (ByronTx atxaux) = fromIntegral txByteSize
+    where
+      txByteSize = (Strict.length . annotation . CC.UTxO.aTaTx      $ atxaux)
```

Comment:
By the way, I'm actually sure if this is correct. I'm thinking that it isn't

Reply:
Because of the tag (`encodeListLen`)?

Reply:
Yeah, exactly. I wanted to avoid hardcoding the bytes required for `encodeListLen 2` (I believe 1 byte in this case?), if possible, so I need to take another look at this.

Reply:
We'll go with a hardcoded solution for now. I have opened an issue for this: https://github.com/input-output-hk/cardano-ledger/issues/576